### PR TITLE
i18n(pt-BR): Update `integrations-reference.md` based on #795 and #808 changes

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description
Updated PT-BR `integrations-reference.md` page based on #795 and #808 changes.

### CODE REVIEW

File: src/pages/pt-BR/reference/integrations-reference.md [hunk 1]
```diff
@@ -265,26 +280,87 @@ O endereço, família e número de porta suprido pelo [módulo Net do NodeJS](ht
...
+
+```js
+import { writeFile } from 'node:fs/promises';
+import { fileURLToPath } from 'node:url';
+export default function minhaIntegracao() {
+  return {
+    hooks: {
+      'astro:build:done': async ({ dir }) => {
+        const metadados = await getIntegrationMetadata();
+        // Utilize fileURLToPath para conseguir uma string de caminho absoluto válida e multiplataforma.
```
Comment (victor0x16) [inline src/pages/pt-BR/reference/integrations-reference.md:307]:
```suggestion
        // Utilize fileURLToPath para conseguir uma string válida e multiplataforma do caminho absoluto.
```

File: src/pages/pt-BR/reference/integrations-reference.md [hunk 2]
```diff
@@ -265,26 +280,87 @@ O endereço, família e número de porta suprido pelo [módulo Net do NodeJS](ht
...
+**Tipo:** [`RouteData[]`](https://github.com/withastro/astro/blob/main/packages/astro/src/%40types/astro.ts#L973)
 
-Um array de todas as rotas geradas. Isso atualmente inclui `pathname` sozinho, apesar de planejarmos incluir metadados no futuro. Nota: isso estará vazio quando estiver utilizando um adaptador de SSR!
+Uma lista de todas as rotas geradas junto de seus metadados associados. **Isso estará vazio quando estiver utilizando um adaptador de SSR!**
 
-#### Opção `dir`
+Você pode ver a referência completa do tipo `RouteData` abaixo, mas as propriedades mais comuns são:
 
-**Tipo:** [`URL`](https://developer.mozilla.org/pt-BR/docs/Web/API/URL)
+- `component` - o caminho do arquivo de entrada relativo a raiz da projeto
```
Comment (victor0x16) [inline src/pages/pt-BR/reference/integrations-reference.md:324]:
```suggestion
- `component` - o caminho do arquivo de entrada relativo à raiz do projeto
```

**Review by victor0x16 — CHANGES_REQUESTED** [review]
Tiny suggestions!

**Comment by Yan-Thomas** [comment]
Thanks for the suggestions and review @victor0x16. Let me know if we are good to merge now!

**Review by victor0x16 — APPROVED** [review]
LGTM.

**Pull request merged**